lexer.py

```
@@ -0,0 +1,100 @@
+        return self.input[ self.position ]
+    
+    def peek( self ) -> str:
+        if int( self.position ) >= len( self.input ) - 1:
+            return end
+        else: 
+            return self.input[ self.position + 1 ]
+    
+    def backup( self ) -> None:
+        self.position -= 1
+
+    def log( self, type: TokenType ) -> None:
```

Flake8 (7.2.0)

[error] whitespace after '('

(E201)